c2corg_api/emails/email_service.py

```diff
@@ -38,6 +38,13 @@ def send_registration_confirmation(self, lang, user, link):
                 subject='Inscription sur Camptocamp.org',
                 body='Pour activer votre compte cliquez sur %s' % link)
 
+    def send_request_change_password(self, lang, user, link):
+        # TODO: handle i18n
+        self._send_email(
+                user.email,
+                subject='Changement de mot de passe sur Camptocamp.org',
```

Comment:
We have an hardcoded prefered language to 'fr' so I think it is best to hardcode sending emails in French. It is a temporary solution until we get https://github.com/c2corg/v6_api/issues/194, hopefully next week.